release-notes/2024-05/README.md

```diff
@@ -0,0 +1,211 @@
+
+# SystemLink Enterprise release 2024-05 Release Notes
+
+The 2024-05 release bundle for SystemLink Enterprise has been published to <https://downloads.artifacts.ni.com>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## New Features and Behavior changes
+
+- Navigate SystemLink Enterprise faster with the intuitively organized UI.
```

Comment:
Please update to: "Navigate SystemLink Enterprise faster with the intuitively organized Home page and the navigation pane."

Reply:
Updated @vpstoynova